repos/system_upgrade/common/actors/scansubscriptionmanagerinfo/tests/test_scansubscriptionmanagementinfo.py

```diff
@@ -0,0 +1,349 @@
+import logging
+import os
+from collections import namedtuple
+
+import pytest
+
+from leapp.libraries import stdlib
+from leapp.libraries.actor import scanrhsm
+from leapp.libraries.stdlib import api
+from leapp.libraries.common import mounting
+from leapp.libraries.common import repofileutils
+from leapp.libraries.common import rhsm
```

Comment:
```suggestion
from leapp.libraries.common import (
    mounting,
    repofileutils,
    rhsm,
)
```

Reply:
Suggested changes were rendered obsolete after moving the introduced tests into the library tests. However, the general idea of not repeating import paths was taken into account and applied when moving the tests.